imqsauth/http.go

```diff
@@ -933,11 +923,11 @@ func httpHandlerSetUserGroups(central *ImqsCentral, w http.ResponseWriter, r *ht
 		}
 	}
 	*/
-	central.broadcastToAllSubscribers(createNotificationMessage(notificationChannel, notificationType, strconv.FormatInt(int64(userId), 10)))
+	central.broadcastToAllSubscribers(createNotificationMessage(strconv.FormatInt(int64(userId), 10)))
 }
 
-func createNotificationMessage(channelName string, typeName string, messageContent string) string {
-	return fmt.Sprintf("%v:%v:%v", channelName, typeName, messageContent)
+func createNotificationMessage(messageContent string) string {
+	return fmt.Sprintf("%v:%v", "permissions_changed", messageContent)
```

Comment:
Why did you remove the typeName? I am just thinking that if we want to send a message with another notification type we either have to change this one or create a new one